utils/babeltrace_thapi.in

```diff
@@ -33,6 +147,10 @@ def common_options(opts)
     $options[:backends] = v
   end
 
+  opts.on('--debug') do |v|
```

Comment:
You didn't add a default value in the default option hash above. I am fine with it but maybe you're not.